### PR TITLE
(maint) Update beaker gems for testing

### PR DESCRIPTION
**Pull request by MikaelSmith**
Updates beaker gems to support setting in CI config. Needed to support
testing macOS 10.12.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,16 +1,14 @@
 version: 1.1.0.{build}
 environment:
-  OPENSSL_INCLUDE_DIR: C:\OpenSSL\include
-  OPENSSL_LIB_DIR: C:\OpenSSL\lib
+  OPENSSL_INCLUDE_DIR: C:\OpenSSL-Win64\include
```
Comment (haus) [inline appveyor.yml]: Did appveyor change or is this us breaking ourself?

Comment (MikaelSmith) [reply appveyor.yml]: We were referring to a package that's no longer hosted (a few lines below). AppVeyor images added 1.0.2 installs of OpenSSL: https://www.appveyor.com/docs/build-environment/#pre-installed-software.

**Comment by MikaelSmith** [comment]
Got some rustc warnings that are causing failure because they're written to stderr.

**Comment by MikaelSmith** [comment]
Got CI working again.

**Pull request merged**